FizzBuzz.py

```diff
@@ -1,27 +1,38 @@
-from types import ListType, IntType
+"""Print all integers from 1 to 100. However, for multiples of three,
+print 'Fizz' instead of the number, for multiples of five, print
+'Buzz', and for numbers which are multiples of both three and five,
+print 'FizzBuzz'.
+"""
+def is_appropriate(number):
+    """Return True if the given number is an integer between 0 and
+    101. Otherwise, return False.
+    """
+    if isinstance(number, int) and 0 < number <= 100:
+        return True
+    return False
 
-def create_list(size):
-    #takes one positive int 'size' and returns a list of all integers 0 to 'size'
-    assert isinstance(size, IntType)
-    some_list = []
-    for i in range(0, size+1):
-        some_list.append(i)
-    return some_list
+def determine_element(element):
+    """Determine whether the element should be named, 'Fizz', 'Buzz',
+    'FizzBuzz', or itself, make the change to the element(if
+    applicable), and return it.
+    """
+    assert is_appropriate(element) is True
```

Comment:
shorten comparisons to true to just the statement, they're equivalent:

assert is_appropriate(element)

also add a message to your assertions, like "Element must be between 1 and 100" so that when they fire, the caller gets something useful